addons/loyalty/static/src/scss/loyalty.scss

```diff
@@ -24,37 +24,17 @@
     }
 }
 
-.loyalty-templates-container {
-    pointer-events: auto;
-
-    .loyalty-template {
-        &, * {
-            transition: all .15s;
-        }
-
-        cursor: pointer !important;
-
-        img {
-            filter: invert(.5);
-        }
-
-        &:hover {
-            * {
-                color: #7C6576 !important;
-            }
-
-            background-color: var(--o-color-4);
-            box-shadow: 0 6px 10px 0 rgba(0, 0, 0, 0.1), 0 2px 2px 0 rgba(0, 0, 0, 0.05);
-            border-color: #7C6576 !important;
-
-            img {
-                filter: invert(0);
-            }
-
-            .card-body {
-                background-color: var(--o-color-4) !important;
-            }
-        }
+.loyalty-template {
+    &:hover {
+        background: var(--LoyaltyTemplate-background-hover, #{$o-gray-200});
+    }
+    &:focus-visible {
+        box-shadow: 0 0 0 #{$btn-focus-width} RGBA(#{$btn-link-focus-shadow-rgb}, .5);
+        outline: none;
+    }
+    &:active {
+        background: #{mix($o-action, $o-gray-100, 15%)};
+        border: $border-width solid #{$o-action};
```

Comment:
Not sure that this part is necessary. It's only visible when you click for such a small amount of time that it looks like it's bugged (same for other screens)

https://tinyurl.com/4y8sdrep